Sources/TripKitUI/views/map annotations/TKUIAnnotations+TripKit.swift

```diff
@@ -153,7 +153,7 @@ extension TKSegment: TKUISemaphoreDisplayable {
         return .headOnly
       }
     } else {
-      return .headWithTime(departureTime, timeZone, isRealTime: timesAreRealTime)
+      return trip.hideExactTimes ? .headOnly : .headWithTime(departureTime, timeZone, isRealTime: timesAreRealTime)
```

Comment:
![Simulator Screen Shot - iPhone 13 - 2021-12-06 at 21 34 57](https://user-images.githubusercontent.com/2722800/144831437-26408bb6-c3a5-4236-a437-716c33529e2e.png)

@nighthawk Can we do this here, assuming hide the text on the `timeLabel` is what is required here.

Reply:
Yep, looks good.